manifests/b/Borvid/HttpMasterProfessional/5.5.0/Borvid.HttpMasterProfessional.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=AUSU.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Borvid.HttpMasterProfessional
+PackageVersion: 5.5.0
+MinimumOSVersion: 10.0.0.0
+InstallerType: msi
+Installers:
+- Architecture: neutral
+  InstallerUrl: https://www.httpmaster.net/downloads/httpmaster_pro_5.5.0.msi
+  InstallerSha256: FB193587C569270417D26D2E0BBCF56886321B482CF2F470F90774D1E6864018
+  ProductCode: '{4FE8DB79-7F6E-4B2B-BBFA-FC7007E6A2AF}'
```

```suggestion
  ProductCode: '{49A2448A-CD0E-4601-83A3-0E7B8EB8C93A}'
```
Must match control panel